execution_workflows/TAPAS/bin/make_tapas_ref.py

```diff
@@ -25,6 +25,6 @@ def readGTF(gtf_path):
  ln=ln.split("\t")
  transcript="NM_"+ln[3]
  gene="NM_"+transcript2gene[ln[3]]
- organized=[gene,transcript,ln[0],ln[5],ln[1],ln[2],ln[6],ln[7],ln[9],ln[10],ln[11]]
+ organized=[gene,transcript,ln[0].strip('chr'),ln[5],ln[1],ln[2],ln[6],ln[7],ln[9],ln[10],ln[11]]
```

Comment:
Had a discussion with @mrgazzara about this since I'm not too familiar with whether it's safe to strip 'chr' or if it's better to have an alternate set of data without 'chr' prefix. I faced a similar issue with Dapars back then because the tool needed 'chr' prefix but our data didn't have it, but we ended up having an alternate test data with 'chr' prefix instead of manually adding 'chr' in dapars worfklow :p I'll refer this to @mrgazzara and @SamBryce-Smith  to review this since I'm not sure! :D

Reply:
I agree with this, I wouldn't remove the 'chr' prefix at all. In past runs with test data I have got output in the BED files with TAPAS (which suggests we don't need to remove it to get TAPAS to run), however this time I was unsuccessful, probably because this meant the chr names in the annotation file no longer matched those in the BAM file

Reply:
I tried running `TAPAS` with the 'chr', which gave me empty outputs, which is why I'm stripping it off as this only outputs an intermediate file.

Reply:
does this mean we should just have an alternate set of data *without* chr prefix for TAPAS? and maybe add in the readme as well that TAPAS needs to run without 'chr' prefix in all the dataset?

Reply:
Now, the workflow requires the input gtf file to have its `chr` prefix stripped